clients/admin-ui/src/features/integrations/configure-monitor/ConfigureMonitorForm.tsx

```diff
@@ -101,10 +115,17 @@ const ConfigureMonitorForm = ({
 
   const initialValues = {
     name: monitor?.name ?? "",
-    execution_start_date: format(Date.now(), "yyyy-MM-dd'T'hh:mm"),
-    execution_frequency: monitor?.execution_frequency,
+    execution_start_date:
+      monitor?.execution_start_date ?? format(Date.now(), "yyyy-MM-dd'T'HH:mm"),
```

Comment:
Slight functional change here, note time format changed from `hh:mm` to `HH:mm` to handle AM/PM correctly.